### ISSUE TITLE
Finetuning for DAG models

### ISSUE THREAD

**Issue by informatica92** [vlfeat/matconvnet#859]
Hi, 
I succesfully performed a finetuning for a non-DAG model:

    ...
    net.layers = net.layers(1:end-2);
    net.layers{end+1} = struct('type', 'conv', ...
    'weights', {{0.05*randn(1,1,prev,new, 'single'), zeros(1,new,'single')}}, ...
    'learningRate', [0.05, 0.05], ...
    'stride', [1 1], ...
    'pad', [0 0 0 0]) ;
    net.layers{end+1} = struct('type', 'softmaxloss') ; 
    ...
    [net, info] = cnn_train(net, imdb, @getbatch);

How can I replace the last layers in a DAG model?

Thanks in advance
Best regards

**Comment by layumi** [comment]
@informatica92  This is my code. I hope it can help you.
For more reference, you can refer to the examples in the /examples/imagenet.
`netStruct = load('./data/imagenet-resnet-50-dag.mat') ;
net = dagnn.DagNN.loadobj(netStruct) ;
net.removeLayer('fc1000');
net.removeLayer('prob');
%---------setting1
for i = 1:numel(net.params)
    if(mod(i,2)==0)
        net.params(i).learningRate=0.02;
    else net.params(i).learningRate=0.001;
    end
    name = net.params(i).name;
    if(name(1)=='b')
        net.params(i).weightDecay=0;
    end
end

%---
net.params(1).learningRate = 0.0001;
dropoutBlock = dagnn.DropOut('rate',0.75);
net.addLayer('dropout',dropoutBlock,{'pool5'},{'pool5d'},{});
fc751Block = dagnn.Conv('size',[1 1 2048 751],'hasBias',true,'stride',[1,1],'pad',[0,0,0,0]);
net.addLayer('fc751',fc751Block,{'pool5d'},{'prediction'},{'fc751f','fc751b'});
net.addLayer('softmaxloss',dagnn.Loss('loss','softmaxlog'),{'prediction','label'},'objective');
net.addLayer('top1err', dagnn.Loss('loss', 'classerror'), ...
    {'prediction','label'}, 'top1err') ;
net.addLayer('top5err', dagnn.Loss('loss', 'topkerror', ...
    'opts', {'topK',5}), ...
    {'prediction','label'}, 'top5err') ;
net.initParams();`

**Comment by micklexqg** [comment]
@informatica92 , have you solved it? 
the same problem...
